### ISSUE TITLE
Add getters to fetch related resources

### ISSUE THREAD

**Issue by Daanvm** [mollie/mollie-api-ruby#42]
Create a nice way to fetch related resources for an object, e.g. all refunds for a payment or all mandates for a customer.

For some relations an entry is added to the `links` property. See the [GET Payments documentation](https://www.mollie.com/en/docs/reference/payments/get) for an example.

For others we should determine the URL ourself, for example by using the customer's ID to fetch the mandates ([documentation](https://www.mollie.com/en/docs/reference/mandates/list)).

**Comment by benoist** [comment]
It might be a good idea to start with adding convenience methods to deal with the resources for example:

```ruby
payment = Payment.create(attributes)
```

This can be a stepping stone towards adding methods for related resources

```ruby
payment = Payment.get(payment[:id])
refund  = payment.refunds.create(attributes)
refunds = payment.refunds.all
refund.delete
```

**Comment by benoist** [comment]
Fixed with #65
